backends/p4tools/modules/testgen/targets/bmv2/backend/protobuf/protobuf.cpp

```diff
@@ -403,7 +403,7 @@ void Protobuf::emitTestcase(const TestSpec *testSpec, cstring selectedBranches,
     LOG5("Protobuf test back end: emitting testcase:" << std::setw(4) << dataJson);
     auto incrementedbasePath = basePath;
     incrementedbasePath.concat("_" + std::to_string(testId));
-    incrementedbasePath.replace_extension(".stf");
+    incrementedbasePath.replace_extension(".proto");
```

Comment:
Thanks, I hit this issue too :(

Reply:
In which context?

Reply:
We were working on the user guide, and observed that the proto backend generated .stf file :)